models/articles.models.js

```diff
@@ -7,4 +7,27 @@ exports.selectArticlesById = (article_id) =>{
         }
         else return rows
         })
-}
\ No newline at end of file
+}
+
+exports.selectAllArticles = () => {
+    const comments = db.query("SELECT * FROM comments;").then(({rows})=> rows)
```

Comment:
Ideally we should be getting this comment_count key from the sql query using an aggregate function. This is a ncie javascript solution you have here but I would suggest changing to use sql only to achieve what you want.